examples/discovery-search-app/cypress/integration/passages/spec.ts

```diff
@@ -90,5 +90,27 @@ describe('Passage Results', () => {
       cy.get('@combinedResult').contains('View passage in document');
       cy.get('@combinedResult').contains('View table in document');
     });
+
+    describe('and clicking on "CI Document" for a result', () => {
```

Comment:
There's a section above: 

`describe('and clicking on "View passage in document" for a result', () => {`

where this code should live. Between the `navigate to` and `close` tests. That is, create another nested `describe` for clicking on the CI tab and running these two tests.